src/patternfly/components/FormControl/examples/FormControl.md

```diff
@@ -123,129 +121,63 @@ import './FormControl.css'
 
 ### Usage
 | Class | Applied to | Outcome |
 | -- | -- | -- |
-| `.pf-v5-c-form-control` | `<input>`,`<textarea>`, `<select>` |  Initiates an input, textarea or select. For styling of checkboxes or radios see the [checkbox component](/components/checkbox) or [radio component](/components/radio). **Required**  |
+| `.pf-v5-c-form-control` | `<div>` |  Initiates a container for an input, textarea or select. For styling of checkboxes or radios see the [checkbox component](/components/checkbox) or [radio component](/components/radio). **Required**  |
+| `.pf-v5-c-form-control` | `<input>`,`<textarea>`, `<select>` |  Initiates an input, textarea or select.  **Required**  |
 | `.pf-m-resize-vertical` | `textarea.pf-m-form-control` | Modifies a `textarea.pf-v5-c-form-control` element so it can only be resized vertically along the y-axis. |
 | `.pf-m-resize-horizontal` | `textarea.pf-m-form-control` | Modifies a `textarea.pf-v5-c-form-control` element so it can only be resized horizontally along the x-axis. |
+| `.pf-m-readonly` | `.pf-v5-c-form-control` | Modifies a form control for a readonly input, textarea, or select.|
+| `.pf-m-disabled` | `.pf-v5-c-form-control` | Modifies a form control for a disabled input, textarea, or select.|
 | `.pf-m-success` | `.pf-v5-c-form-control` | Modifies a form control for the success state. |
 | `.pf-m-warning` | `.pf-v5-c-form-control` | Modifies a form control for the warning state. |
-| `.pf-m-icon-sprite` | `.pf-v5-c-form-control` | Modifies form control element to use an external SVG sprite instead of embedded data URIs for icons. For use with apps whose content security policies disallow the use of data URIs. |
-| `.pf-m-icon` | `input.pf-v5-c-form-control` | Modifies a form control text input to be able to specify a custom SVG background via `--pf-v5-c-form-control--m-icon--BackgroundUrl`, and other optional vars for other background properties.
-| `.pf-m-calendar` | `.pf-v5-c-form-control.pf-m-icon` | Modifies a form control to support the calendar icon. |
-| `.pf-m-clock` | `.pf-v5-c-form-control.pf-m-icon` | Modifies a form control to support the clock icon. |
-| `.pf-m-expanded` | `input.pf-v5-c-form-control` | Modifies a form control for the expanded state. This is used when clicking in the text input toggles something open/closed. |
-| `.pf-m-placeholder` | `select.pf-v5-c-form-control` | Modifies a form select for placeholder styles. This modifier is set programatically based on the chosen option. |
-| `.pf-m-plain` | `input[readonly].pf-v5-c-form-control`, `textarea[readonly].pf-v5-c-form-control` | Modifies an `<input>` or `<textarea>` with a `readonly` attribute to be presented as normal text. |
+| `.pf-m-error` | `.pf-v5-c-form-control` | Modifies a form control for the error (invalid) state. |
+| `.pf-v5-c-form-control__icon` | `div` | Creates a container for an icon associated with a text input.
```

Comment:
Looks like `.pf-v5-c-form-control__utilities` is missing from the docs, same for `pf-m-status` and `pf-m-icon`. One other nit:

```suggestion
| `.pf-v5-c-form-control__icon` | `div` | Creates a container for an icon associated with a form control.
```